spring-cloud-gcp-pubsub/src/test/java/com/google/cloud/spring/pubsub/support/DefaultSubscriberFactoryTests.java

```diff
@@ -40,6 +42,9 @@
 	@Mock
 	private CredentialsProvider credentialsProvider;
 
+	@Mock
+	private PubSubConfiguration pubSubConfiguration;
```

Comment:
You'll want to add tests for all kinds of configuration state -- what kind of executor provider ends up provided for a default set of properties, for a subscription-specific set of properties.
This is also where the "oops, what happens if someone uses the deprecated constructor" tests go -- I sometimes use tests to figure out what _should_ happen.